sms2jwplayer/test/test_genupdatejob.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import unittest
+
+from sms2jwplayer.genupdatejob import convert_acl
+
+LOG = logging.getLogger(__name__)
+
+
+class ConvertAclTests(unittest.TestCase):
+    """All tests for :py:`~genupdatejob.convert_acl`"""
+
+    def test_null_acls(self):
+        """Check in particular that [''] is treated as a null ACL"""
+
+        # null ACLs
+        self.assertEqual(convert_acl('world', None), 'WORLD')
+        self.assertEqual(convert_acl('world', ''), 'WORLD')
+        self.assertEqual(convert_acl('world', []), 'WORLD')
+        self.assertEqual(convert_acl('world', ['']), 'WORLD')
+
+        # non-null ACL
+        self.assertEqual(convert_acl('world', ['102592']), 'GROUP_102592')
+
+    def test_visibility_acl_combinations(self):
+        """
+        Test conversion for all combinations (found in PROD data) of media item visibility and acl
+        """
+
+        # check that INST, GROUP, & USER ACE's convert
+        self.assertEqual(
+            convert_acl('acl-overrule', ['001705', 'UIS', 'mjd66', 'aj333']),
+            'GROUP_001705,INST_UIS,USER_mjd66,USER_aj333'
```

Comment:
Add upper-case CRSids and, since we also see them in the data-base I think, lower-case instids.